src/modules/tests/test-data/cat-b/test-requirements/test-requirements.reducer.ts

```diff
@@ -17,3 +18,6 @@ export function testRequirementsReducer(
       return state;
   }
 }
+
+export const getTestRequirementsCatB =
+  createFeatureSelector<CatBUniqueTypes.TestRequirements>('testRequirements');
```

Comment:
is the naming of the file itself in keeping with the convention. Should it not be test-requirements.cat-b.reducer.ts?

Reply:
Correct it's wrong, I've got a bit of work on the side to look at the Test Requirements as there is confusion in the app between them and the Legal Requirements. I'll do the rename in that bit of work.